dev/Dockerfile

```diff
@@ -29,26 +29,40 @@ ENV PATH "${DENO_INSTALL}/bin:${PATH}"
 
 COPY --from=deno-bin /usr/bin/deno /bin/deno
 
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends \
-    # base
-    git \
-    curl \
-    python3 \
-    make \
-    cmake \
-    # grpc
-    protobuf-compiler \
-    libprotobuf-dev \
-    # openssl
-    pkg-config \
-    libssl-dev \
-    # wasmedge
-    libclang-dev \
-    && rm -rf /var/lib/apt/lists/* \
-    && curl -fsS https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh -o wasmedge.sh \
-    && bash wasmedge.sh -v $WASMEDGE_VERSION \
-    && rm wasmedge.sh
+RUN set -eux; \
+    export DEBIAN_FRONTEND=noninteractive; \
```

Comment:
Nah, it's just best practices in case some of the installers have prompts that require user input. Deets [here](https://askubuntu.com/questions/972516/debian-frontend-environment-variable).